packages/jmix-react-core/src/app/Auth.ts

```diff
@@ -0,0 +1,76 @@
+import {createJmixServerError, JmixServerError} from "./JmixServerError";
+import {base64encode} from "../util/base64";
+
+export interface CommonAuthOptions {
+  endpoint: string;
+  clientId: string;
+  secret: string;
+  locale?: string;
+}
+
+export interface LoginOptions extends CommonAuthOptions {
+  userName: string;
+  password: string;
+}
+
+export interface LogoutOptions extends CommonAuthOptions {
+  token: string;
+}
+
+export function login(options: LoginOptions): Promise<{ access_token: string }> {
+  const {
```

Comment:
From feedback of users - it would be nice to have an ability override fetchOptions (e.g. headers)